.github/workflows/ci_jruby.yml

```diff
@@ -123,22 +49,24 @@ jobs:
         uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # tag v3.5.0
 
       - name: Install JRuby
-        uses: ruby/setup-ruby@bc1dd263b68cb5626dbb55d5c89777d79372c484 # tag v1.151.0
+        uses: ruby/setup-ruby@036ef458ddccddb148a2b9fb67e95a22fdbf728b # tag v1.160.0
         with:
-          ruby-version: jruby-9.4.3.0
+          ruby-version: jruby-9.4.5.0
 
       - name: Bundle
         run: bundle install
 
       - name: Run Multiverse Tests
-        run: ./.github/workflows/scripts/retry_command 
+        uses: nick-fields/retry@943e742917ac94714d2f408a0e8320f2d1fcafcd # tag v2.8.3
+        with:
+          timeout_minutes: 20
+          max_attempts: 3
+          command: "bundle exec rake test:multiverse[${{ matrix.suite }}]"
         env: 
-          TEST_CMD: "bundle exec rake test:multiverse[group=${{ matrix.multiverse }}]"
           VERBOSE_TEST_OUTPUT: true
-          RETRY_ATTEMPTS: 5
```

Comment:
What's your thinking behind removing the retry attempts? I apologize if this has already been discussed!